### PR TITLE
[PM-384] Fix zero projects crashing the all projects page

### PR DESCRIPTION
**Pull request by MattHallEA**

### CODE REVIEW

File: app/models/pafs_core/state.rb [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # frozen_string_literal: true
 module PafsCore
   class State < ActiveRecord::Base
+    VALID_STATES = %w[draft completed submitted updatable updated archived finished].freeze
+
     belongs_to :project, inverse_of: :state
-    validates :state, inclusion: { in: %w[draft completed submitted updatable updated archived finished] }
+    validates :state, inclusion: { in: VALID_STATES }
```
Comment (baphled) [inline app/models/pafs_core/state.rb]: Could we move this in to a separate PR :)

**Comment by baphled** [comment]
Love that we're refactoring though I'd likely do it as a separate PR :)

File: app/models/pafs_core/state.rb (hunk 1 above)
Comment (MattHallEA) [reply app/models/pafs_core/state.rb]: Yeah - this ended up here as I found that the endpoint wasn't being tested. I'll move it to a separate PR. Do I need a separate JIRA ticket too @baphled?

Comment (baphled) [reply app/models/pafs_core/state.rb]: Nah, diff PR should be cool :)

**Comment by MattHallEA** [comment]
OK, refactor moved to #232
